aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -120,5 +120,74 @@ Tensor prod(const Tensor &self, int64_t dim_, bool keepdim) {
 }
 
 // \DIM REDUCE ################################################################
+
+// MULTI DIM REDUCE ###########################################################
+
+Tensor sum(const Tensor &self, IntList dims_, bool keepdim) {
```

Comment:
It would be great if we could have a templatized version of this, so we could easily expand this to cover more of our reductions!